packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -472,7 +490,22 @@ export class DeltaManager
         return DeltaManager.detailsFromConnection(connection);
     }
 
-    public async connectCore(args: IConnectionArgs = {}): Promise<IDocumentDeltaConnection> {
+    /**
+     * Start the connection. Any error should result in container being close.
+     * And report the error if it excape for any reason.
+     * @param args - The connection arguments
+     */
+    private triggerConnect(args: IConnectionArgs) {
+        this.connectCore(args).catch((err) => {
+            // Errors are raised as "error" event and close container.
+            // Have a catch-all case in case we missed something
+            if (!this.closed) {
+                this.logger.sendErrorEvent({ eventName: "ConnectException" }, err);
```

Comment:
I'd not log anything (and ideally make connectCore void, i.e. fire and forget).
DeltaManager already reports properly all the errors.
The only reason connectCore() returns anything is to support deprecated creation-on-connection flow that should be removed from code base.

Reply:
Ah, it's moved code...

Reply:
yep... we can't clean it up later, but I am reusing existing code.